portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,5 +28,6 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
     response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<p>Hello Ryan!!</p>");
```

Comment:
should this replace "Hello world"?

Reply:
I'm not sure. The documentation just wanted us to make sure we knew how to access the /data page and edit it.

Reply:
I'll replace it just so it looks nicer.